site/content/docs/5.2/getting-started/parcel.md

```diff
@@ -1,101 +1,149 @@
 
-[Install bootstrap]({{< docsref "/getting-started/download#npm" >}}) as a Node.js module using npm.
+<br>
 
-Bootstrap depends on [Popper](https://popper.js.org/), which is specified in the `peerDependencies` property. This means that you will have to make sure to add both of them to your `package.json` using `npm install @popperjs/core`.
+1. **Create a project folder and setup npm.** We'll create the `my-project` folder and initialize npm with the `-y` argument to avoid it asking us all the interactive questions.
 
-When all will be completed, your project will be structured like this:
+   ```sh
+   mkdir my-project && cd my-project
+   npm init -y
+   ```
 
-```text
-project-name/
-├── build/
-├── node_modules/
-│   ├── @popperjs/
-|   |   └── core/
-│   └── bootstrap/
-├── scss/
-│   └── custom.scss
-├── src/
-│   ├── index.html
-│   └── index.js
-└── package.json
-```
+2. **Install Parcel.** Next we need to install our Parcel development dependencies: .
 
-## Importing JavaScript
+   ```sh
+   npm i --save-dev parcel
+   ```
 
-Import [Bootstrap's JavaScript]({{< docsref "/getting-started/javascript" >}}) in your app's entry point (usually `src/index.js`). You can import all our plugins in one file or separately if you require only a subset of them.
+3. **Install Bootstrap.** Now we can install Bootstrap. We'll also install Popper since our dropdowns, popovers, and tooltips depend on it for their positioning. If you don't plan on using those components, you can omit Popper here.
 
-<!-- eslint-skip -->
-```js
-// Import all plugins
-import * as bootstrap from 'bootstrap';
+   ```sh
+   npm i --save bootstrap @popperjs/core
+   ```
 
-// Or import only needed plugins
-import { Tooltip as Tooltip, Toast as Toast, Popover as Popover } from 'bootstrap';
+## Project structure
 
-// Or import just one
-import Alert as Alert from '../node_modules/bootstrap/js/dist/alert';
-```
+We've already created the `my-project` folder and initialized npm. Now we'll also create our `src` folder, stylesheet, and JavaScript file to round out the project structure.
 
-## Importing CSS
+1. **Create the project structure.** Run the following from `my-project`, or manually create the folder and file structure shown below.
 
-To utilize the full potential of Bootstrap and customize it to your needs, use the source files as a part of your project's bundling process.
+   ```sh
+   mkdir {src,src/js,src/scss}
+   touch src/index.html src/js/main.js src/scss/styles.scss src/scss/_custom.scss
+   ```
 
-Create your own `scss/custom.scss` to [import Bootstrap's Sass files]({{< docsref "/customize/sass#importing" >}}) and then override the [built-in custom variables]({{< docsref "/customize/sass#variable-defaults" >}}).
+   When you're done, your complete project should look like this:
 
-## Build app
+   ```text
+   my-project/
+   ├── src/
+   │   ├── js/
+   │   │   └── main.js
+   │   ├── scss/
+   │   │   ├── _custom.scss
```

Comment:
In Webpack guide and this one, we create this `_custom.scss` file that we don't use. Shoud we keep it?